floris/tools/optimization/scipy/yaw.py

```diff
@@ -294,40 +371,47 @@ def reinitialize_opt(
                 If none are specified, default values of
                 {'std_wd': 4.95, 'std_yaw': 1.75, 'pmf_res': 1.0,
                 'pdf_cutoff': 0.995} are used. Defaults to None.
+            turbine_weights (iterable, optional): weighing terms that allows the
+                user to emphasize power gains at particular turbines or
+                completely ignore power gains from other turbines. The array
+                of turbine powers from floris is multiplied with this array in
+                the calculation of the objective function. Defaults to an array
+                with all values 1.0 and length equal to the number of turbines.
             calc_init_power (bool, optional): If True, calculates initial wind
                 farm power for each set of wind conditions. Defaults to True.
         """
         if minimum_yaw_angle is not None:
             self.minimum_yaw_angle = minimum_yaw_angle
         if maximum_yaw_angle is not None:
             self.maximum_yaw_angle = maximum_yaw_angle
+        self.x_baseline = [
+                turbine.yaw_angle
+                for turbine in self.fi.floris.farm.turbine_map.turbines
+        ]
         if x0 is not None:
             self.x0 = x0
+            if not all(np.array(x0) == np.array(self.x_baseline)):
+                print("WARNING: Baseline yaw angle in FLORIS differ from initial optimization conditions (self.x0)")
```

Comment:
I don't think this needs to be a warning. For example, I imagine it would be very common for the baseline values to be zero, but the initial guess for optimal offsets to be some other value.